src/magaz/CommonBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,3 @@
+services:
+    common.abstract:
+        class: magaz\CommonBundle\AbstractController
```

Comment:
Please move full class name to parameter
```
parameters:
    common.abstract.class: magaz\CommonBundle\AbstractController
```

and use this parameter here as `%common.abstract.class%`